script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_2B.txt

```diff
@@ -90,9 +90,9 @@ Human flesh and blood are also a source of nutrition for servants, but most of i
 The Castle Lord accumulates power as they sleep in their coffin, eventually turning the entire community into their own flesh and blood.
 ―――That is the nature of the vampire that lurks in this city."
 <Page28>
-"At the moment, they don't have all that many servants, so they're not that formidable. However, as the number of victims increases, so too does the power of a Castle Lord.
-I was hoping to find their coffin and kill them before that happens, but I still have yet to locate the enemy's resting place.
-This vampire has hidden themselves very skillfully, I can't sense their presence at all."
+"At the moment, they don't have all that many servants and so they're not that formidable. However, as the number of victims increases, so too does the power of a Castle Lord.
```

Comment:
**the** castle lord

Reply:
I think either might work here? It's generally true of castle lords, but also true of this specific one

Reply:
I read it as speaking in generalities since she specifies castle lord.

Reply:
I wanted to keep consistent with line 90. Don't mind either way